nss-day-cohort-30/bangazon-site-mighty-mahi-mahi
# Abandoned Product Type Report

Fixes:
-Added an abandoned product type report that shows 5 most commonly occurring product types in open orders

Changes proposed in this request & reasons behind organizational or architectural decisions:
-Created AbandonedProductType view model to use in AbandonedProductType report

Steps to test:
-Start Bangazon
-Click reports in nav bar
-Click on abandoned product types in report list
-Check to see if the number of products in open orders is listed by product type

System configuration (3rd party libraries to be installed, command line utilities to run, UI instructions, expected outcome):
-N/A

Commit message:
-abandoned products report now displays a list of product types and the number of products of that type that are in open orders
- abandoned product types report now only shows top 5 most abandoned product types
--

Link to feature ticket:
-https://github.com/nss-day-cohort-30/bangazon-site-mighty-mahi-mahi/issues/11

@mighty-mahi-mahi
